src/management-system-v2/components/processes.tsx

```diff
@@ -67,15 +66,19 @@ const { Search } = Input;
 const Processes: FC = () => {
   const router = useRouter();
 
-  const { data, isLoading, isError, isSuccess } = useGetAsset('/process', {});
+  const { data, isLoading, isError, isSuccess } = useGetAsset('/process', {
+    params: {
+      query: { noBpmn: true },
+    },
+  });
 
   const setProcesses = useProcessesStore((state) => state.setProcesses);
   const setSelectedProcess = useProcessesStore((state) => state.setSelectedProcess);
 
   const [open, setOpen] = useState(false);
 
   const [selection, setSelection] = useState<Processes>([]);
-  const [hovered, setHovered] = useState<Process | undefined>(undefined);
+  const [hovered, setHovered] = useState<ApiData<'/process', 'get'>[number] | undefined>(undefined);
```

Comment:
It wouldn't really be cleaner since data is of type `Process[] | undefined` so we would need to exclude the undefiend and get type of element of the list, it would look something like this `Exclude<typeof data, undefined>[number]|undefined`